src/Components/Facility/ConsultationDetails.tsx

```diff
@@ -14,228 +14,348 @@ import { DailyRoundsModel } from "../Patient/models";
-    } else if (dailyRoundsListData.length > 0) {
-        roundsList = dailyRoundsListData.map((itemData, idx) => {
-            return (
-                <div key={`daily_round_${idx}`} className="w-full mt-4 px-2">
-                    <div className="block border rounded-lg bg-white shadow h-full cursor-pointer hover:border-primary-500 text-black">
-                        <div className="p-4">
-                            <Grid container justify="space-between" alignItems="center">
-                                <Grid item xs={11} container spacing={1}>
-                                    <Grid item xs={6}>
-                                        <Typography>
-                                            <span className="text-gray-700">Temperature:</span>{" "}
-                                            {itemData.temperature}
-                                        </Typography>
-                                    </Grid>
-                                    <Grid item xs={6}>
-                                        <Typography>
-                                            <span className="text-gray-700">Taken at :</span>{" "}
-                                            {itemData.temperature_measured_at ? moment(itemData.temperature_measured_at).format('lll') : "-"}
-                                        </Typography>
-                                    </Grid>
-                                    <Grid item xs={12}>
-                                        <Typography>
-                                            <span className="text-gray-700">Physical Examination Info:</span>{" "}
-                                            {itemData.physical_examination_info}
-                                        </Typography>
-                                    </Grid>
-                                    <Grid item xs={12}>
-                                        <Typography>
-                                            <span className="text-gray-700">Other Details:</span>{" "}
-                                            {itemData.other_details}
-                                        </Typography>
-                                    </Grid>
-                                </Grid>
-                            </Grid>
-                            <div className="mt-2">
-                                <Button
-                                    size="small"
-                                    variant="outlined"
-                                    fullWidth
-                                    onClick={(e) => navigate(`/facility/${facilityId}/patient/${patientId}/consultation/${consultationId}/daily-rounds/${itemData.id}`)}
-                                >View Daily Rounds Details</Button>
-                            </div>
-                        </div>
-                    </div>
-                </div >
-            );
-        });
-    }
+  let roundsList: any;
```

Comment:
Codacy found an issue: ['roundsList' is already defined.](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5808160)